docs/runbook/Features.md

```diff
@@ -1,145 +1,70 @@
 # Features
 
-The following table indicates how different features impact the result.
```

Comment:
Why is this table removed?

Reply:
I don't believe the table is particularly accurate. There are too many variables involved in many cases.

Such a table might be useful to give a user a quick way to scan what feature to check out. I'm just not sure what it should look like.